src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(BallColors.values().length);
+        return BallColors.values()[index].toString();
```

Comment:
Instead of using `toString()`, you should use the `name()` method to get the String representation of the enum constants. This ensures that you get the correct name even if the `toString()` method is overridden in the enum.